### PR TITLE
Fix template interpolation in vfor when readonly array

### PR DESCRIPTION
**Pull request by yoyo930021**
When array is readonly, I will get `any` in `v-for` element.

### CODE REVIEW

File: server/src/services/typescriptService/bridge.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export declare const ${componentHelperName}: {
 };
 export declare const ${iterationHelperName}: {
   <T>(list: T[], fn: (value: T, index: number) => any): any;
+  <T>(list: readonly T[], fn: (value: T, index: number) => any): any;
```
Comment (ktsn) [inline server/src/services/typescriptService/bridge.ts]: I believe just changing existing `T[]` overload is enough rather than adding a new one.

Comment (yoyo930021) [reply server/src/services/typescriptService/bridge.ts]: I change `T[]` to `readonly T[] | T[]`.

Comment (ktsn) [reply server/src/services/typescriptService/bridge.ts]: I mean just changing it to `readonly T[]` because it's super type of `T[]`

Comment (yoyo930021) [reply server/src/services/typescriptService/bridge.ts]: Wow! I'm done.

**Pull request merged**